[2025-09-13T09:41:07.013+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-13T09:41:07.307+0000] {taskinstance.py:2066} INFO - Dependencies not met for <TaskInstance: dbt_marts.marts.fct_order_details.run manual__2025-09-13T09:40:29.757092+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2025-09-13T09:41:07.329+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
